Block/Catalog/Product/View.php

```diff
@@ -104,11 +104,13 @@ private function getProduct()
     private function getPlans()
     {
         foreach ($this->config->getPaymentPlansConfig()->getEnabledPlans() as $planConfig) {
-            $this->plans[] = array(
-                'installmentsCount' => $planConfig->installmentsCount(),
-                'minAmount' => $planConfig->minimumAmount(),
-                'maxAmount' => $planConfig->maximumAmount()
-            );
+            if( in_array($planConfig->installmentsCount(), array(2,3,4,10,12)) ){
```

Comment:
I think it is not the plugin job to filter `installmentsCount`


why did you perform this test ?